packages/asc-ui/src/components/Alert/AlertStyle.ts

```diff
@@ -41,6 +41,9 @@ export const ContentWrapper = styled.div`
   flex-direction: column;
   justify-content: center;
   width: 100%;
+  overflow-wrap: anywhere;
+  overflow: hidden;
+  margin-right: ${themeSpacing(4)};
```

Comment:
I don't think we will need this margin, as it's will cause problems on smaller screens:
![image](https://user-images.githubusercontent.com/2486227/125056864-f4231700-e0a8-11eb-923e-b102ba4635fb.png)
Versus current styling:
![image](https://user-images.githubusercontent.com/2486227/125056977-0c933180-e0a9-11eb-99a3-aeb5cb2ee215.png)

To be honest, I think we should refactor this component so the close button will float right and the test will wrap around the close button. I've tried to change the position of the close button in the HTML (next to the header / children), gave it a `float: right` and removed the styling from it's parent (that had `display: flex`). This way the paragraph can also use the full width!
![image](https://user-images.githubusercontent.com/2486227/125057786-e5892f80-e0a9-11eb-91b3-c73453558ed5.png)
I'm not sure if this will cause any other problems yet tho, but this could be a potential solution and would kill 2 birds with one stone.

Reply:
Thanks @chrisvanmook for your feedback (love it!) and your patience! I had a week off. ;) I changed the PR. The `overflow-wrap: anywhere;` is needed for a long contiguous text but is not supported for Safari on IOS. I added the `overflow: hidden;` for these specific cases.